lib/sass/calcite-web/patterns/pagination.scss

```diff
@@ -0,0 +1,35 @@
+// ┌────────────┐
+// │ Pagination │
+// └────────────┘
+//  ↳ http://esri.github.io/calcite-web/documentation/patterns/#pagination
+//  ↳ patterns → _pagination.md
+
+@mixin pagination() {
+  padding: $baseline;
+  display: flex;
+  align-items: center;
+
+  svg {
```

Comment:
I would rather give this its own class like `pagination-arrow` or something. We try not to attach styles to raw html elements if we can help it.

If it has it's own class, you can also move it outside this mixin/give it it's own mixin.